src/test/java/games/strategy/test/EqualsPredicateTests.java

```diff
@@ -179,7 +186,8 @@ public void shouldReturnFalseWhenOwnerObjectsAreNotEqual() {
     }
   }
 
-  private static final class FakeClass {
+  @Nested
+  private final class FakeClass {
```

Comment:
This class does not contain tests, and thus does not need to be annotated with `@Nested`.

It also can remain `static` because it's a child of the top-level class.